src/components/ClientQuote/ClientQuote.tsx

```diff
@@ -0,0 +1,37 @@
+import styles from "./ClientQuote.module.scss";
+import finishLine from "../../assets/images/finishLine.jpg";
+import Image from "next/image";
+import classNames from "classnames";
+import { Quote } from "../Quote/Quote";
+import { SwiperQuotes } from "../SwiperQuotes/SwiperQuotes";
+
+export function ClientQuote() {
+  return (
+    <div className={styles.main}>
+      <SwiperQuotes />
+
+      <div className={styles.imageWrapper}>
+        <div className={classNames(styles.mobileImage, styles.image)}>
+          <Image
+            src={finishLine}
+            objectFit="cover"
+            alt="Sam Pausha crossing the finish line at the Des Moines Ironman"
```

Comment:
you can assign this alt to a constant before the export on line 7. that way you only have to change it once if you need to edit

`const HERO_IMG_ALT = "Sam Pausha crossing the finish line at the Des Moines Ironman"`